Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1573,101 +1610,132 @@ namespace xr
                     }
                 }
 
-                // Get gamepad data 
+                // Get interaction data based on input profile
+                if (!InputSources[idx].interactionProfileName.empty())
                 {
```

Comment:
nit: its a good idea to test the controllers as well to make sure they are not broken with this change :)

Reply:
I've tested in simulation and can see the select gesture going through. Do you have any specifics to test?

Reply:
I see, that should be fine but for the sake of sanity, could you test all the functionalities like trigger, thumbstick and trackpad as well?

Reply:
I tested motion controllers with simulation, and hit the expected breakpoints in WebXRControllerComponent.prototype.update for when a button or axis update, so the controller data is being piped through properly still.